msvc/libmonoruntime.targets

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)monozlib.targets" />

```

Comment:
Can I just inline the ItemGroup here?

Reply:
In theory yes, but it would be good to keep it in the file, so it can be shared with other project configurations going forward. Any specific reason why you would do that?

Reply:
Fewer files is nice. We have so many in this directory. But ok.